Source/Core/Core/PowerPC/GDBStub.cpp

```diff
@@ -309,10 +309,18 @@ static void HandleQuery()
 {
   DEBUG_LOG_FMT(GDB_STUB, "gdb: query '{}'", CommandBufferAsString() + 1);
 
-  if (!strcmp((const char*)(s_cmd_bfr + 1), "TStatus"))
-  {
-    return SendReply("T0");
-  }
```

Comment:
Did you mean to get rid of this?  It seems to be a valid packet ([`qTStatus`](https://sourceware.org/gdb/onlinedocs/gdb/Tracepoint-Packets.html#index-qTStatus-packet))

Reply:
I did because TStatus is about a feature called tracepoint which we do not support.

Reply:
Ah, I guess it does make sense to remove it then.